### DISCUSSION TITLE
hub to hub communication via wire?

### DISCUSSION THREAD

**Discussion by Potato-eater** [pybricks/support#1200] [Q&A]
hi, i have already seen the [bluetooth hub to hub communication](https://pybricks.com/projects/tutorials/wireless/hub-to-hub/broadcast/) page. I am just wondering, can this be achieved with a double head wire?

**Comment by laurensvalk** [comment]
This is not enabled in Pybricks, but it could be when https://github.com/pybricks/support/issues/220 is done. And you'd need a double head cable with:
- GND connected on both ends.
- TX/RX swapped so they can talk to each other.
- Everything else disconnected. You don't want the motor power terminals firing at one another.
